03_array.ipynb

```diff
@@ -597,7 +590,11 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
```

Comment:
The answer is not hidden, as it should be.

![image](https://user-images.githubusercontent.com/17503044/100468018-88ec7d80-30a1-11eb-9890-7b807bbbd030.png)

Reply:
It does not seem to look any different to me?

Reply:
Looks like this change is only visible in JupyterLab, not in a _regular_ Jupyter notebook.

Currently it looks like this
![image](https://user-images.githubusercontent.com/17503044/100496776-cfcf8700-3124-11eb-8308-5e9a3788b7be.png)

After the change it looks like this
![image](https://user-images.githubusercontent.com/17503044/100496786-de1da300-3124-11eb-9533-caec339b1b6b.png)

Reply:
OK, well it's designed to be run in jlab, so that's fine.